cms/jinja2/templates/base.html

```diff
@@ -63,26 +66,13 @@
         }
     },
     "footer": {
-        "rows": [
-            {
-                "itemsList": [
-                    {
-                        "text": 'Contact us',
-                        "url": '#0'
-                    },
-                    {
-                        "text": 'Cookies and privacy',
-                        "url": '#0'
-                    }
-                ]
-            }
-        ],
-        "oglLink": {
-            "pre": 'All content is available under the',
-            "text": 'Open Government Licence v3.0',
-            "url": 'https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/',
-            "post": ', except where otherwise stated'
-        }
+        "cols": footer_menu_columns(footer_menu),
+    "oglLink": {
+        "pre": 'All content is available under the',
+        "text": 'Open Government Licence v3.0',
+        "url": 'https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/',
+        "post": ', except where otherwise stated'
```

Comment:
let's mark the pre and post strings as translatable

```suggestion
        "pre": _('All content is available under the'),
        "text": 'Open Government Licence v3.0',
        "url": 'https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/',
        "post": _(', except where otherwise stated')
```

see the footer on https://cy.ons.gov.uk/

Reply:
✅ [1067893](https://github.com/ONSdigital/dis-wagtail/pull/111/commits/1067893aa8ff7870288467daf7a4a8d5b3ba8869)